server/services/team/service.ts

```diff
@@ -1,4 +1,5 @@
 import { Db } from 'mongodb';
+import { Meteor } from 'meteor/meteor';
```

Comment:
we should avoid importing Meteor on services, as doing so we cannot run this service standalone